### ISSUE TITLE
Release parquet 1.11.1

### ISSUE THREAD

**Issue by asfimport** [apache/parquet-java#2445]
Some issues are discovered during the migration to the parquet-mr release 1.11.0 in Spark. These issues are to be fixed and release in the minor release 1.11.1.

**Reporter**: [Gabor Szadovszky](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=gszadovszky) / @gszadovszky
**Assignee**: [Gabor Szadovszky](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=gszadovszky) / @gszadovszky
#### Related issues:
- [Make ParquetFileReader.getFilteredRecordCount public](https://issues.apache.org/jira/browse/PARQUET-1740) (is blocked by)
- [Some filters throws ArrayIndexOutOfBoundsException](https://issues.apache.org/jira/browse/PARQUET-1744) (is blocked by)
- [No result for partition key included in Parquet file](https://issues.apache.org/jira/browse/PARQUET-1745) (is blocked by)
- [Changed the data order after DataFrame reuse](https://issues.apache.org/jira/browse/PARQUET-1746) (is blocked by)
- [Invalid filteredRowCount in InternalParquetRecordReader](https://issues.apache.org/jira/browse/PARQUET-1765) (is blocked by)
- [Minimize the parquet-avro fastutil shaded jar](https://issues.apache.org/jira/browse/PARQUET-1853) (is blocked by)

<sub>**Note**: *This issue was originally created as [PARQUET-1774](https://issues.apache.org/jira/browse/PARQUET-1774). Please see the [migration documentation](https://issues.apache.org/jira/browse/PARQUET-2502) for further details.*</sub>

**Comment by asfimport** [comment]
[Dongjoon Hyun](https://issues.apache.org/jira/browse/PARQUET-1774?#comment-17021327) / @dongjoon-hyun:
@gszadovszky. Could you add `is blocked by` links to those required issues?

**Comment by asfimport** [comment]
[H. Vetinari](https://issues.apache.org/jira/browse/PARQUET-1774?#comment-17039253):
@gszadovszky @dongjoon-hyun
All blockers of the release of 1.11.1 seem to have been finished (PARQUET-1739 hasn't been closed yet, but all subtasks seem to have been completed) - anything else that stands in the way of a release?

**Comment by asfimport** [comment]
[Gabor Szadovszky](https://issues.apache.org/jira/browse/PARQUET-1774?#comment-17039845) / @gszadovszky:
Waiting for Spark to confirm that 1.11.1-SNAPSHOT works properly.

**Comment by asfimport** [comment]
[Matthew McMahon](https://issues.apache.org/jira/browse/PARQUET-1774?#comment-17142883) / @maccamlc:
Hi

Is 1.11.1 something that might be released soon?

I just came across quite a big issue when using 1.11.0 (when using Map type) and reported as PARQUET-1879

I have also put together a PR if someone can review, as imo it would be great if its possible to include in the next release.

Regards
Matt

**Comment by asfimport** [comment]
[Gabor Szadovszky](https://issues.apache.org/jira/browse/PARQUET-1774?#comment-17143031) / @gszadovszky:
Hi @maccamlc,

Originally 1.11.1 was planned for Spark integration. As this effort seems to be blocked (because of Avro update) it seems fair to postpone the potential Spark related issues to 1.11.2 (or even 1.12) and release 1.11.1 with the current fixes and PARQUET-1879 in addition. But let's fix that issue first. :)

Cheers,
Gabor

**Comment by asfimport** [comment]
[Matthew McMahon](https://issues.apache.org/jira/browse/PARQUET-1774?#comment-17143310) / @maccamlc:
Thanks @gszadovszky, sounds good to me :)

**Comment by asfimport** [comment]
[Gabor Szadovszky](https://issues.apache.org/jira/browse/PARQUET-1774?#comment-17184049) / @gszadovszky:
Missed to close this one after the release. Now, it is done.
